quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/rest/TimefoldProcessorHotReloadTest.java

```diff
@@ -27,11 +28,11 @@ public class TimefoldProcessorHotReloadTest {
     @Test
     void solverConfigHotReload() {
```

Comment:
This test (and its `MultipleSolvers` counterpart) seems to throw a lot of exceptions into the sysout, even though it is passing. Let's fix that.